src/main/resources/templates/db_scripts/relatedInstanceType.sql

```diff
@@ -0,0 +1,53 @@
+DROP FUNCTION IF EXISTS set_related_instance_type_md_json;
+
+CREATE FUNCTION set_related_instance_type_md_json ()
+RETURNS trigger AS $$
+  BEGIN
+    IF NEW.creation_date IS NULL THEN
+      RETURN NEW;
+    end IF;
+
+    NEW.jsonb = jsonb_set(NEW.jsonb, '[metadata, createdDate]', to_jsonb(NEW.creation_date));
+    IF NEW.created_by IS NULL THEN
+      NEW.jsonb = NEW.jsonb #- '{metadata, createdByUserId}';
+    ELSE
+      NEW.jsonb = jsonb_set(NEW.jsonb,'{metadata, createdByUserId}' to_jsonb(NEW.created_by));
+    END IF;
+    RETURN NEW;
+  END;
+$$ LANGUAGE plpgsql COST 100
+GO
```

Comment:
`GO` is a psql command, it is not available when running these db_scripts during migration.